UefiTestingPkg/Include/Library/FlatPageTableLib.h

```diff
@@ -104,29 +104,54 @@ CreateFlatPageTable (
   IN OUT PAGE_MAP  *Map
   );
 
+/**
+  Dumps the contents of the input PAGE_MAP to the debug log.
+**/
+VOID
+EFIAPI
+DumpPageMap (
+  IN PAGE_MAP  *Map
+  );
+
 /**
   Checks the input flat page/translation table for the input region and converts the associated
-  table entries to EFI access attributes (EFI_MEMORY_XP, EFI_MEMORY_RO, EFI_MEMORY_RP). If the
-  access attributes vary across the region, EFI_NO_MAPPING is returned.
-
-  @param[in]  Map                 Pointer to the PAGE_MAP struct to be parsed
-  @param[in]  Length              Length in bytes of the region
-  @param[in]  Length              Length of the region
-  @param[out] Attributes          EFI Attributes of the region
-
-  @retval EFI_SUCCESS             The output Attributes is valid
-  @retval EFI_INVALID_PARAMETER   The flat translation table has not been built or
-                                  Attributes was NULL or Length was 0
-  @retval EFI_NOT_FOUND           The input region could not be found
-  @retval EFI_NO_MAPPING          The access attributes are not consistent across the region.
+  table entries to EFI access attributes (EFI_MEMORY_XP, EFI_MEMORY_RO, EFI_MEMORY_RP). The caller
+  of this function is responsible for checking ActualCheckedLength if the return value is
+  EFI_NOT_FOUND or EFI_NO_MAPPING. EFI_NOT_FOUND indicates that the attributes vary across
+  the region. EFI_NO_MAPPING indicates that the section from RegionStart to RegionStart +
+  ActualCheckedLength is not mapped. This implies that if ActualCheckedLength == RegionLength,
+  the entire input region is not mapped.

```

Comment:
```suggestion
  the region. EFI_NO_MAPPING indicates that the section from RegionStart to RegionStart +
  ActualCheckedLength is not mapped. If ActualCheckedLength == RegionLength,
  when EFI_NO_MAPPING is returned, the entire input region is not mapped.
```

nit: suggesting for clarity and a stronger statement

Reply:
Updated